tests/configlet/util/run_test_in_switch.py

```diff
@@ -0,0 +1,117 @@
+#! /usr/bin/env python
```

Comment:
There is a directory for GCU tests in sonic-mgmt, it is `tests/generic_config_updater`, do you think it would be good to move the test to this directory to have everything in a single place?

Reply:
AddRack -- Scenario based test.
It does it via configlet and now enhanced to use generic updater. Both configlet & via generic updater only differ on the tool/command  used for application and shares the rest of the code  like test preparation & result analysis, which is a big bulk and yet Add/remove rack specific.
Hence stays out.

Going ahead, the generic_config_updater will be like another common "sudo config ..." usage. More & more  tests will use  generic-config updater as a part, but test would cover more than that and hence stay outside generic-config-updater folder.

Reply:
Sounds good.